telco-core/configuration/reference-crs-kube-compare/required/networking/Network.yaml

```diff
@@ -0,0 +1,12 @@
+# required
+# count: 1
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  defaultNetwork:
+    ovnKubernetesConfig:
+      gatewayConfig:
+        routingViaHost: true
```

Comment:
@imiller0  routingViaHost: false is the ovnk default. Silly question, is routingViaHost something that we *enforce* in the RDS?  I know it's here https://docs.openshift.com/container-platform/4.14/scalability_and_performance/telco_ref_design_specs/core/telco-core-ref-crs.html#networking-yaml_ran-core-ref-design-crs  but I'm just wondering

Reply:
Yes, Telco only supports and mandates OVN local gateway mode.

Reply:
+1 on Carlos' reply :)

Reply:
thanks!